.github/CHANGELOG.md

```diff
@@ -465,8 +489,39 @@
 
 <h3>Improvements</h3>
 
+* The `group_observables` transform is now differentiable.
+  [(#1483)](https://github.com/PennyLaneAI/pennylane/pull/1483)
+ 
+  For example:
+
+  ``` python
+  import jax
+  from jax import numpy as jnp
+  
+  coeffs = jnp.array([1., 2., 3.])
```

Comment:
Awesome, we should include examples from different interfaces like this in our docs more often :100: